run

```diff
@@ -432,7 +432,15 @@ build_local() {
 
 copy_gds_files_() {
     local root_folder=${1:-${TOP}}
-    local cufile_search="${root_folder}/temp/cuda/include:${root_folder}/temp/cuda/lib64"  #"/usr/local/cuda/include:/usr/local/cuda/lib64 ${PREFIX:-}/include:${PREFIX:-}/lib ${CONDA_PREFIX:-}/include:${CONDA_PREFIX:-}/lib ${root_folder}/temp/cuda/include:${root_folder}/temp/cuda/lib64"
+    local cufile_search="${root_folder}/temp/cuda/include:${root_folder}/temp/cuda/lib64"
+    if [[ ! -z "${CONDA_PREFIX+x}" ]]
+    then
+      cufile_search="${CONDA_PREFIX:-}/include:${CONDA_PREFIX:-}/lib ${cufile_search}"
+    fi
+    if [[ ! -z "${PREFIX+x}" ]]
+    then
+      cufile_search="${PREFIX:-}/include:${PREFIX:-}/lib ${cufile_search}"
+    fi
```

Comment:
Noticed we are setting the paths to search for cuFile here, changed to add `$PREFIX` & `$CONDA_PREFIX`  before the paths already included